python/graphstorm/dataloading/dataloading.py

```diff
@@ -722,11 +777,19 @@ class GSgnnNodeDataLoader():
         the device trainer is running on.
     train_task : bool
         Whether or not for training.
+    reconstructed_embed_ntype : list of str
+        The node types that requires to construct node features.
     """
-    def __init__(self, dataset, target_idx, fanout, batch_size, device, train_task=True):
+    def __init__(self, dataset, target_idx, fanout, batch_size, device, train_task=True,
+                 reconstructed_embed_ntype=None):
         self._data = dataset
         self._fanout = fanout
         self._target_nidx  = target_idx
+        if reconstructed_embed_ntype is None:
+            reconstructed_embed_ntype = []
+        self._reconstructed_embed_sampler = \
+                _ReconstructedNeighborSampler(dataset, reconstructed_embed_ntype, 5) \
```

Comment:
Can anyone tune the fanout of _ReconstructedNeighborSampler?

Reply:
we don't allow tuning the fanout right now. we should allow tuning.